_posts/2015-06-18-a-man-a-plan-a-canal-a-savas.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "A man a plan a canal a... Savas!"
+date: 2015-06-18
+author: Chris Russo
+tags: business identity
+summary: The truth behind the palindrome.
+---
+
+Previously, on our [who we are](/team) page, I gave you a taste of the 
+[meaning](/team#what-does-savas-mean) of Savas. Now let's take a deeper dive
```

Comment:
Need some punctuation at the end of this line. Might I recommend a few exclamation points?!?!